rules/evasion/fake-process.yara

```diff
@@ -42,3 +43,17 @@ rule fake_systemd : critical {
   condition:
 	filesize < 100MB and $ref
 }
+
+rule fake_process_names : high {
+	meta:
+		description = "mentions known fake process names"
+	strings:
+		$kdevchecker = "kdevchecker" fullword
+		$kworkerr = "kworkerr" fullword
+		$ksoftriqd = "ksoftriqd" fullword
+		$kdevtmpfsi = "kdevtmpfsi" fullword
+		$kthreaddk = "kthreaddk" fullword
+		$deamon = "deamon" fullword
```

Comment:
Should this be `daemon` in both cases?

Reply:
Nope - this is an intentional misspelling based on what I found in malware samples.